models/airdrop.go

```diff
@@ -23,6 +23,48 @@ type AirdropClaim struct {
 	Claimed       bool    `json:"claimed" gorm:"not null" sql:"DEFAULT:0"`
 }
 
+func GetAirdrop(airdrop *Airdrop) Airdrop {
+	return Airdrop{
+		ID:           airdrop.ID,
+		TokenID:      airdrop.TokenID,
+		MerkleRoot:   airdrop.MerkleRoot,
+		RawData:      airdrop.RawData,
+		OnChainIndex: airdrop.OnChainIndex,
+	}
+}
```

Comment:
This is a bit redundant. Gorm automatically returns an airdrop struct. You'd only want to do this if you don't want to return some values and not others.